lib/cash-in/cash-in-tx.js

```diff
@@ -2,7 +2,7 @@ const _ = require('lodash/fp')
 const pgp = require('pg-promise')()
```

Comment:
This whole file was changed only because of automatic formatting that's not really configured and of the blacklist file rename. You can undo all changes here.